app/Http/Controllers/Api/ApiController.php

```diff
@@ -64,13 +64,17 @@ public function item($data, $code = 200, $meta = [], $transformer = null)
      * @param  array  $meta
      * @return \Illuminate\Http\JsonResponse
      */
-    public function transform($data, $code = 200, $meta = [])
+    public function transform($data, $code = 200, $meta = [], $include = null)
```

Comment:
actually...that might not work if you have `$meta` then you will want to pass it. hmmm. Again, I'm not too bothered by it, but will let others way in.